updates/3/updates.sql

```diff
@@ -70,11 +70,73 @@ INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x5273063725a43
 *
 INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xd794DD1CAda4cf79C9EebaAb8327a1B0507ef7d4", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xd794DD1CAda4cf79C9EebaAb8327a1B0507ef7d4", "C", "HYVE", "HYVE", 1602545820, NULL, "hyve", "", "0xd794DD1CAda4cf79C9EebaAb8327a1B0507ef7d4");
 *
 UPDATE assets SET cryptocompare="CPC" WHERE identifier="_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090";/*Update crytocompare*/
-INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", "C", "CPChain", "CPC", 1516060800, NULL, "cpchain", "CPC", "0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090");
\ No newline at end of file
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090", "C", "CPChain", "CPC", 1516060800, NULL, "cpchain", "CPC", "0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090");
+UPDATE assets SET coingecko="amo" WHERE identifier="_ceth_0x38c87AA89B2B8cD9B95b736e1Fa7b612EA972169";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x38c87AA89B2B8cD9B95b736e1Fa7b612EA972169", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x38c87AA89B2B8cD9B95b736e1Fa7b612EA972169", "C", "AMO Coin", "AMO", 1522540800, NULL, "amo", "AMO", "0x38c87AA89B2B8cD9B95b736e1Fa7b612EA972169");
+UPDATE assets SET coingecko="blockchain-certified-data-token" WHERE identifier="_ceth_0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5";/*Update coingecko*/
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5", "C", "Blockchain Certified Data Token", "BCDT", 1513735057, NULL, "blockchain-certified-data-token", "BCDT", "0xAcfa209Fb73bF3Dd5bBfb1101B9Bc999C49062a5");
+UPDATE assets SET coingecko=NULL WHERE identifier="_ceth_0xAef38fBFBF932D1AeF3B808Bc8fBd8Cd8E1f8BC5";/*Update coingecko*/
```

Comment:
Why are you doing this? The coin is just inactive: https://www.coingecko.com/en/coins/creditbit

The identifier **should stay**. We can still query icon and/or historical price data.

Just add it to the delisted assets here: https://github.com/rotki/rotki/blob/ea64757937e0f23fdeb16c98c2cf4b64567c6be7/rotkehlchen/externalapis/coingecko.py#L37